### PR TITLE
[BugFix] update storage_cache_ttl default 30 days

### PR DESCRIPTION
**Pull request by abc982627271**
## What type of PR is this：
- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Specify default storage_cache_ttl when creating lake table is 30 days. If you set it to be 0 or you don't set it, fe would transfer this value to be 30 days.

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [x] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

### CODE REVIEW

**Review by gengjun-git — DISMISSED** [review]

File: fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java [hunk 1]
```diff
@@ -2043,8 +2043,9 @@ private void createOlapOrLakeTable(Database db, CreateTableStmt stmt) throws Ddl
                 if (!enableStorageCache && storageCacheTtlS != 0) {
                     throw new DdlException("Storage cache ttl should be 0 when cache is disabled");
                 }
+                // default ttl is 30 days
                 if (enableStorageCache && storageCacheTtlS == 0) {
-                    storageCacheTtlS = Config.tablet_sched_storage_cooldown_second;
+                    storageCacheTtlS = PropertyAnalyzer.DEFAULT_STORAGE_CACHE_TTL;
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java:2048]: throw exception?
"Storage cache ttl should not be 0 when cache is enabled"

File: fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java [hunk 2]
```diff
@@ -54,6 +54,7 @@
 
 
 public class PropertyAnalyzer {
+    public static final long DEFAULT_STORAGE_CACHE_TTL = 2592000L;
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java:57]: put it in config like `tablet_sched_storage_cooldown_second`?

**Comment by abc982627271** [comment]
run starrocks_fe_unittest

**Comment by abc982627271** [comment]
run starrocks_fe_unittest

**Comment by abc982627271** [comment]
run starrocks_fe_unittest

**Comment by dulong41** [comment]
run all

**Comment by dulong41** [comment]
run starrocks_admit_test

**Comment by dulong41** [comment]
run starrocks_admit_test

**Comment by dulong41** [comment]
run starrocks_clang-format

**Comment by dulong41** [comment]
run starrocks_be_unittest

**Comment by dulong41** [comment]
run starrocks_clang-tidy

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 3 / 5 (60.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|1|2|50.00%|[2052]|
|:large_blue_circle:|com/starrocks/sql/ast/SingleRangePartitionDesc.java|1|2|50.00%|[151]|
|:large_blue_circle:|com/starrocks/common/Config.java|1|1|100.00%|[]|

**Review by gengjun-git — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 3 / 5 (60.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|1|2|50.00%|[2053]|
|:large_blue_circle:|com/starrocks/sql/ast/SingleRangePartitionDesc.java|1|2|50.00%|[151]|
|:large_blue_circle:|com/starrocks/common/Config.java|1|1|100.00%|[]|

**Review by gengjun-git — APPROVED** [review]

**Review by wyb — APPROVED** [review]

**Pull request merged**